prow/BUILD.bazel

```diff
@@ -54,13 +54,13 @@ filegroup(
     srcs = [
         ":package-srcs",
         "//prow/apis/prowjobs:all-srcs",
-        "//prow/artifact-uploader:all-srcs",
+        "//prow/artifactuploader:all-srcs",
```

Comment:
We should not change binary and/or image names in this PR. This will break users.

Reply:
Ok, Will revert naming